### ISSUE TITLE
dropdown:aria-expanded missing

### ISSUE THREAD

**Issue by michaelkushlan** [pluralsight/design-system#1958]
## Instructions

- fill in title: format of "<package name>: short description"
- choose: "bug" or "enhancement request" below, fill it in, and delete other section

## Enhancement Request
Add aria-expanded="true" when dropdown is open and aria-expanded="false" or removed when dropdown is closed

### Today's behavior
No aria-expanded attribute

Problems/friction you encounter.
Accessibility audit for Verizon TQ

### The Value Add
Accessibility audit for Verizon TQ

Related WAI docs: https://www.w3.org/TR/wai-aria-practices-1.1/examples/listbox/listbox-collapsible.html
